sda/cmd/ingest/ingest_test.go

```diff
@@ -67,11 +85,21 @@ func (suite *TestSuite) TestTryDecrypt_wrongFile() {
 	buf, err := io.ReadAll(file)
 	assert.NoError(suite.T(), err)
 
-	key, err := config.GetC4GHKey()
-	assert.Nil(suite.T(), err)
-	b, err := tryDecrypt(key, buf)
-	assert.Nil(suite.T(), b)
-	assert.EqualError(suite.T(), err, "not a Crypt4GH file")
+	keyList, err := config.GetC4GHprivateKeys()
+	assert.NoError(suite.T(), err)
+
+	var decryptionSuccessful bool
+	for _, key := range keyList {
+		b, err := tryDecrypt(key, buf)
+		if b != nil || err == nil {
+			decryptionSuccessful = true
+
+			break
+		}
+		assert.EqualError(suite.T(), err, "not a Crypt4GH file")
+	}
+
+	assert.False(suite.T(), decryptionSuccessful, "Decryption should not succeed with any key")
```

Comment:
```suggestion
	b, err := tryDecrypt(key[0], buf)
	assert.EqualError(suite.T(), b, nil)
	assert.EqualError(suite.T(), err, "not a Crypt4GH file")
```

Taking in the scope of this particular test, this is the only thing that is needed.

Reply:
Fixed in b81ce66c